### ISSUE TITLE
RT Tweet

### ISSUE THREAD

**Issue by rsssxd** [j-fbriere/squawker#10]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/j-fbriere/squawker/releases).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/j-fbriere/squawker/issues) or [closed](https://github.com/j-fbriere/squawker/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug, that only occurs in Squawker

### Affected version

v3.1.3

### Steps to reproduce the bug

Find a tweet that was RTed

### Expected behavior

Quotations of RT tweets appear to be broken.

### Actual behavior

Quotations in RT tweets should be displayed properly.

### Screenshots/Screen recordings

![Screenshot_20230716-093941_Squawker](https://github.com/j-fbriere/squawker/assets/60405617/e7b7336f-f983-4d28-8e42-4c368e77666d)
![Screenshot_20230716-093952_Squawker](https://github.com/j-fbriere/squawker/assets/60405617/5d75efbb-3b50-427b-81dd-b30d8d51d867)

### Affected Android/Custom ROM version

LineageOS A10

### Affected device model

Redmi K20

**Comment by j-fbriere** [comment]
Just to be clear.
When you write that the text of a retweet is broken, do you mean the end of the text with twitter.com/... ?

**Comment by rsssxd** [comment]
The link at the end is another tweet content. However, it looks corrupted because of RT.
![IMG_20230716_101431_089](https://github.com/j-fbriere/squawker/assets/60405617/bfe4be2e-a41c-4c0c-ab1e-40d1fc99b7e9)
![IMG_20230716_101447_302](https://github.com/j-fbriere/squawker/assets/60405617/0c30a266-d087-4f25-a92f-704e9204c706)

**Comment by j-fbriere** [comment]
Version [3.1.4](https://github.com/j-fbriere/squawker/releases/tag/v3.1.4) should fix the issue.

**Comment by rsssxd** [comment]
https://github.com/j-fbriere/squawker/assets/60405617/1dbc7ad6-9dd2-4866-8526-bede71649ad7

**Comment by rsssxd** [comment]
the problem continues

**Comment by j-fbriere** [comment]
Maybe you should flush the feed cache by scroll down from the top.

If that doesn't work, please provide a twitter user id with the problem with a recent tweet, I will test it.

**Comment by rsssxd** [comment]
I cleared the cache and tried again and it was fine. Thanks.

**Comment by j-fbriere** [comment]
I'm glad that everything is fine.

I did some more tests with the **asaf__agah** user but this user has problems.
Now the feeds with this user returns empty response: no tweet, no retweet, nothing.
I tested it on **nitter**: same problem.

This do not apply for profile timeline request, only feed request.

Since this user has problems with feed that do not look related to the current issue, I will close the issue.
